backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/humanaethica/config/HEPermissionEvaluator.java

```diff
@@ -56,6 +56,15 @@ public boolean hasPermission(Authentication authentication, Object targetDomainO
                     Assessment assessment = assessmentRepository.findById(id).orElse(null);
                     if (assessment == null) return false;
                     return assessment.getVolunteer().getId().equals(((Volunteer)authUser.getUser()).getId());
+                case "PARTICIPATION.VOLUNTEER":
+                    Participation participation1 = participationRepository.findById(id).orElse(null);
+                    if (participation1 == null) return false;
+                    return participation1.getVolunteer().getId().equals(((Volunteer)authUser.getUser()).getId());
+                case "PARTICIPATION.CREATOR":
```

Comment:
Check this code, because PARTICIPATION.CREATOR is for creation, not for the member to submit a rating